components/table/index.css

```diff
@@ -12,529 +12,440 @@
+		&:focus:hover {
+			--spectrum-table-icon-color: var(--highcontrast-table-icon-color-focus, var(--mod-table-icon-color-focus-hover, var(--spectrum-table-icon-color-focus-hover)));
+		}
 
-		/* Collapsible and Thumbnails */
-		--mod-table-thumbnail-block-spacing: var(--mod-table-thumbnail-block-spacing-spacious, var(--spectrum-table-thumbnail-to-top-minimum-small-spacious));
-		--mod-table-thumbnail-size: var(--mod-table-thumbnail-size-spacious, var(--spectrum-thumbnail-size-300));
+		&:focus-visible,
+		&.is-keyboardFocused {
+			--spectrum-table-icon-color: var(--highcontrast-table-icon-color-focus, var(--mod-table-icon-color-key-focus, var(--spectrum-table-icon-color-key-focus)));
+		}
 	}
 
-	&.spectrum-Table--sizeL {
-		/* Size and Spacing */
-		--mod-table-min-row-height: var(--mod-table-min-row-height--spacious, var(--spectrum-table-row-height-large-spacious));
-		--mod-table-row-top-to-text: var(--mod-table-row-top-to-text--spacious, var(--spectrum-table-row-top-to-text-large-spacious));
-		--mod-table-row-bottom-to-text: var(--mod-table-row-bottom-to-text--spacious, var(--spectrum-table-row-bottom-to-text-large-spacious));
-
-		/* Row Selection */
-		--mod-table-row-checkbox-block-spacing: var(--mod-table-row-checkbox-block-spacing--spacious, var(--spectrum-table-row-checkbox-to-top-large-spacious));
-
-		/* Collapsible and Thumbnails */
-		--mod-table-thumbnail-block-spacing: var(--mod-table-thumbnail-block-spacing-spacious, var(--spectrum-table-thumbnail-to-top-minimum-large-spacious));
-		--mod-table-thumbnail-size: var(--mod-table-thumbnail-size-spacious, var(--spectrum-thumbnail-size-700));
+	/* These styles get applied to the headCell when the headCell has the --menuButton modifier class. */
+	&.spectrum-Table-menuButton {
+		--spectrum-table-cell-inline-spacing: 0;
+		--spectrum-table-header-row-top-to-text: 0;
+		--spectrum-table-header-row-bottom-to-text: 0;
 	}
+}
 
-	&.spectrum-Table--sizeXL {
-		/* Size and Spacing */
-		--mod-table-min-row-height: var(--mod-table-min-row-height--spacious, var(--spectrum-table-row-height-extra-large-spacious));
-		--mod-table-row-top-to-text: var(--mod-table-row-top-to-text--spacious, var(--spectrum-table-row-top-to-text-extra-large-spacious));
-		--mod-table-row-bottom-to-text: var(--mod-table-row-bottom-to-text--spacious, var(--spectrum-table-row-bottom-to-text-extra-large-spacious));
-
-		/* Row Selection */
-		--mod-table-row-checkbox-block-spacing: var(--mod-table-row-checkbox-block-spacing--spacious, var(--spectrum-table-row-checkbox-to-top-extra-large-spacious));
-
-		/* Collapsible and Thumbnails */
-		--mod-table-thumbnail-block-spacing: var(--mod-table-thumbnail-block-spacing-spacious, var(--spectrum-table-thumbnail-to-top-minimum-extra-large-spacious));
-		--mod-table-thumbnail-size: var(--mod-table-thumbnail-size-spacious, var(--spectrum-thumbnail-size-800));
-	}
+.spectrum-Table-scroller {
```

Comment:
Leaving this here because I don't have another great place to put it and I can't find where @5t3ph wrote her comment about this earlier:

Is this issue with quiet & scrollable addressed in Part 2?

![image](https://github.com/user-attachments/assets/27c3886c-2fd7-4969-a905-a47ca1bcff87)

Reply:
Yes- in part 2 the background color of the header columns gets updated so this doesn't happen.